### PR TITLE
Add Groonga::IndexCursor.min_enable?/min_enable=

### PR DESCRIPTION
**Pull request by komainu8**

### CODE REVIEW

**Comment by komainu8** [comment]
@kenhys Could you review this PR, please?

File: test/test-index-cursor.rb [hunk 1]
```diff
@@ -155,6 +155,11 @@ def test_term
     assert_equal("l", term.key)
   end
 
+  def test_set_min_enable
+    Groonga.min_enable = true
+    assert_equal(true, Groonga.min_enable)
+  end
```
Comment (kenhys) [inline test/test-index-cursor.rb]: How about tlike this? (check get/set at once)

```ruby
def test_min_enable
   default_value = Groonga.min_enable
   Groonga.min_enable = true
   assert_equal([false, true]
                         [default_value, Groonga.min_enble])
end
```

**Comment by komainu8** [comment]
@kenhys Thank you for your comment.
I've fixed it.

**Comment by komainu8** [comment]
@kou Could you review this PR, please?

**Comment by komainu8** [comment]
@kou Thank you for your comments.
I've fixed them.

File: test/test-index-cursor.rb [hunk 2]
```diff
@@ -155,6 +156,15 @@ def test_term
     assert_equal("l", term.key)
   end
 
+  def test_set_min_enable
```
Comment (kou) [inline test/test-index-cursor.rb]: `_enable` is needless.

File: ext/groonga/rb-grn-index-cursor.c [hunk 3]
```diff
@@ -160,4 +174,9 @@ rb_grn_init_index_cursor (VALUE mGrn)
 
     rb_define_method(rb_cGrnIndexCursor, "next", rb_grn_index_cursor_next, 0);
     rb_define_method(rb_cGrnIndexCursor, "each", rb_grn_index_cursor_each, -1);
+
+    rb_define_method(rb_cGrnIndexCursor, "set_min",
```
[the remark this answers is not in the document]
Comment (kou) [reply ext/groonga/rb-grn-index-cursor.c]: Could you use `set_min?`?
We should use `?` for predicate.

**Comment by komainu8** [comment]
@kou Thank you for your comments.
I've fixed them.

**Pull request merged**